python/tk_multi_loader/dialog.py

```diff
@@ -352,6 +348,30 @@ def __init__(self, action_manager, parent=None):
         show_details = self._settings_manager.retrieve("show_details", False)
         self._set_details_pane_visiblity(show_details)
 
+        ################################################
+        # set up welcome window
+        highlight_color = self.palette().highlight().color().name()
+        welcome_msg = (
+            "Welcome! Learn more about the Loader App <a "
+            "href='https://help.autodesk.com/view/SGDEV/ENU/?guid=SG_Supervisor_Artist_sa_integrations_sa_integrations_user_guide_html#the-loader' "
+            "style='text-decoration: none;'><font color='%s'>here.</font></a>"
+            % highlight_color
```

Comment:
Can we use the format function instead? I find it easier to visualize variable on long strings.
```python
welcome_msg = (
    "Welcome! Learn more about the Loader App <a href='{url}' "
    "style='text-decoration: none;'><font color='{color}'>here</font></a>."
    "".format(
        color=highlight_color,
        url=documentation_url
    )
)
```